### PR TITLE
LPS-193943 Error message for maximum number of characters for name does not informs which input is not attending the requirement

### PR DESCRIPTION
**Pull request by larissacribeiro**
Related Issue: [LPS-193943](https://liferay.atlassian.net/browse/LPS-193943)
Related to this: https://github.com/liferay-objects/liferay-portal/pull/2557 
Re-sending with small change after Brian's comment here: https://github.com/brianchandotcom/liferay-portal/pull/140899

### CODE REVIEW

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java [hunk 1]
```diff
@@ -49,9 +50,9 @@ public static class MustBeLessThan75Characters
 
 		public MustBeLessThan75Characters() {
 			super(
-				Collections.singletonList(75),
+				Arrays.asList(75, "external-reference-code"),
```
Comment (marcelabc) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java:53]: Did you guys test if the key `external-reference-code` is displayed as a word, not a key?

**Comment by marcelabc** [comment]
ci:test:relevant

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java (hunk 1 above)
Comment (marcelabc) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java:53]: @larissacribeiro @murilostodolni

Comment (larissacribeiro) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java:53]: Yes, @marcelabc, I did! It's being displayed like this:

erc_lenght_error

Comment (marcelabc) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java:53]: Thanks @larissacribeiro !

**Comment by marcelabc** [comment]
I analyzed the CI results, and the unique failures are not related.
Do you prefer to send directly to Brian? Or do you prefer that we run again the CI to get a new result?

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by larissacribeiro** [comment]
Thanks, Marcela. Maybe we can try running just once more, to see if the stable passes?

**Comment by larissacribeiro** [comment]
Sure @murilostodolni, I will do that now.

**Comment by larissacribeiro** [comment]
ci:forward:force

**Pull request closed without merging**